src/repl/repl_annotations.cpp

```
@@ -394,25 +419,30 @@ void process_str_replace(const StrReplaceAction& action, std::istream& in, std::
         pclose(pipe);
         out << "[copied to clipboard]\n";
       }
-      return;
+      return "";
     }
     if (answer == "?") {
       out << "  y = apply this change\n";
       out << "  n = skip this change\n";
       out << "  t = trust — auto-approve all remaining changes this session\n";
       out << "  c = copy new content to clipboard\n";
+    } else if (answer[0] == 'n' || answer[0] == 'N') {
```

Clang (14.0.6)

[note] +1, nesting level increased to 2

(clang)